radix-engine-tests/tests/blueprints/account_locker.rs

```diff
@@ -1418,32 +1418,28 @@ fn state_of_the_account_locker_can_be_reconciled_from_events_alone() {
             )
         });
 
+    fn set_disallowed_preference(builder: ManifestBuilder, account: ComponentAddress, resource: ResourceAddress) -> ManifestBuilder {
```

Comment:
Could also turn this into an extension trait if you want to retain the builder pattern. For example,
```rust
trait ManifestBuilderExt {
  fn set_disallowed_preference(self, account: ComponentAddress, resource: ResourceAddress) -> Self;
}

impl ManifestBuilderExt for ManifestBuilder {
  fn set_disallowed_preference(self, account: ComponentAddress, resource: ResourceAddress) -> Self {
      self.call_method(
        account,
        ACCOUNT_SET_RESOURCE_PREFERENCE_IDENT,
        AccountSetResourcePreferenceInput {
            resource_address: resource,
            resource_preference: ResourcePreference::Disallowed,
        }
    )
  }
}
```

Reply:
Yup! I almost did it at the time, but thought for one test it was a bit much.

But I’ll do it because it’s fun 🫡